modules/calcite/src/test/sql/subquery/scalar/test_complex_correlated_subquery.test_ignore

```diff
@@ -6,7 +6,7 @@
 # Ignore https://issues.apache.org/jira/browse/IGNITE-15593
 # Ignore https://issues.apache.org/jira/browse/IGNITE-15588
 # Ignore https://issues.apache.org/jira/browse/IGNITE-15592
-# Ignore https://issues.apache.org/jira/browse/IGNITE-15603
+# Ignore https://issues.apache.org/jira/browse/IGNITE-16000
```

Comment:
Why not with IGNITE-16000 ?

Reply:
The new problem was identified, so I've created a new ticket and added information to this test.